backend/project-management-system/src/main/java/org/egov/works/web/controllers/ProjectApiController.java

```diff
@@ -38,17 +49,27 @@ public ProjectApiController(ObjectMapper objectMapper, HttpServletRequest reques
 
     @RequestMapping(value = "/v1/_create", method = RequestMethod.POST)
     public ResponseEntity<ProjectResponse> projectV1CreatePost(@ApiParam(value = "Details for the new Project.", required = true) @Valid @RequestBody ProjectRequest project) {
```

Comment:
we should not have V1 in the method names. That is defined in the path itself. This should just be createProject.

Reply:
Method name changed to createProject